docs/input.md

```diff
@@ -90,6 +98,10 @@ The parent process' input can be re-used in the subprocess by passing `'inherit'
 await execa({stdin: 'inherit'})`npm run scaffold`;
 ```
 
+> [!NOTE]
+> Be careful when combining `input` with `stdin: 'inherit'` or `stdio: 'inherit'`;
+> See [this section](#combining-input-with-stdin-inherit) for details.
```

Comment:
For concision, let's assume users read this file in its entirety. If so, they will read that section below, so we do not need to point to it from those 3 sections above (and also the 2 places in `api.md`). This will also help keep this documentation page concise.